src/test/unit/ilp.test.js

```diff
@@ -37,7 +41,8 @@ describe('ILP', () => {
     });
 
 
-    test('ILP packet should contain a valid transaction object', () => {
+    // TODO: this test has no assertions
+    test.skip('ILP packet should contain a valid transaction object', () => {
```

Comment:
if there is no assertion, there is probably an indirect test of throwing the exception from `Buffer.from` https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_string_encoding